website/source/partials/builder/openstack/_RunConfig-required.html.md

```diff
@@ -0,0 +1,62 @@
+<!-- Code generated from the comments of the RunConfig struct in builder/openstack/run_config.go; DO NOT EDIT MANUALLY -->
+
+-   `source_image` (string) - The ID or full URL to the base image to use. This is the image that will
+    be used to launch a new server and provision it. Unless you specify
+    completely custom SSH settings, the source image must have cloud-init
+    installed so that the keypair gets assigned properly.
+    
+-   `source_image_name` (string) - The name of the base image to use. This is an alternative way of
+    providing source_image and only either of them can be specified.
+    
+-   `source_image_filter` (ImageFilter) - Filters used to populate filter options. Example:
+    
+    ``` json {
+        "source_image_filter": {
+            "filters": {
+                "name": "ubuntu-16.04",
+                "visibility": "protected",
+                "owner": "d1a588cf4b0743344508dc145649372d1",
+                "tags": ["prod", "ready"],
+                "properties": {
+                    "os_distro": "ubuntu"
+                }
+            },
+            "most_recent": true
+        }
+    } ```
+    
+    This selects the most recent production Ubuntu 16.04 shared to you by
+    the given owner. NOTE: This will fail unless *exactly* one image is
+    returned, or `most_recent` is set to true. In the example of multiple
+    returned images, `most_recent` will cause this to succeed by selecting
+    the newest image of the returned images.
+    
+    -   `filters` (map of strings) - filters used to select a
```

Comment:
Oh yes, github is confused, I think because I put the triple back tick at the end of the line and not on the new line the highlighting started to be confused. I hat to tweak those around so that it looks ok on the website.

Reply:
Should be fixed now 🙂